src/shared/com/example/model/timezone.cljc

```diff
@@ -612,7 +612,12 @@
    :WET                              "WET"
    :Zulu                             "Zulu"})
 
-(def us-zone-names (filterv #(str/starts-with? % "US/") (vals time-zones)))
+(def us-zone-ids
```

Comment:
I forgot I did that...I think back when it wasn't an autocomplete and caused a huge list...probably could use all zones now?

Reply:
Sure thing.

Reply:
Thanks!